server/services/property.service.js

```diff
@@ -216,3 +216,13 @@ export const searchThroughProperties = async ({ state, city, houseType, minPrice
       },
     },
   ]);
+
+export const getDistinctPropertyStatesAndTypes = async () => {
```

Comment:
We should use a more explicit name so that the method name still makes sense if we update the fields based on future suggestions.
```suggestion
export const getAvailablePropertyOptions = async () => {
```